docs/04-documentation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update components documentation directly in GitHub
+
+## Step 0: GitHub account
+
+If you don't already have a GitHub account, make sure to create one and log on with it.
+
+## Step 1: Open documentation file
+
+* Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
+* Locate the component you wish to edit. Components are located in `framework/components`
+* Open the README.md file in that component (example: `framework/components/ecl-blockquotes/README.md`)
+
+## Step 2: Edit the documentation
+
+* Click on the edit button (pencil icon on top right)
+* Make any modification you wish by using markdown editor. Here is a [cheatsheet for markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+* You can see result directly by switching to the "Preview" tab
+
+## Step 3: Send your modifications
+
+* Once you have modified the file, it should be sent to us. To do so, still on the edit page, scroll down to the "Commit changes" form
```

Comment:
`it should be sent to us` sounds uncertain, whereas `a notification will be sent to us` for sure :)

Reply:
Indeed, "should" is not correct. My idea was more something like "it has to be sent to us", as it requires some action from the person editing the page (i.e. filling the form)